Tribler/Main/Build/Win/setuptribler.py

```diff
@@ -59,12 +59,15 @@
     # (Disabling bundle_files for now -- apparently causes some issues with Win98)
     # options = {"py2exe": {"bundle_files": 1}},
     # zipfile = None,
+    # py2exe includes some dlls which it should not see:
+    # http://stackoverflow.com/questions/20930173/how-to-include-components-of-psutil-in-py2exe-that-py2exe-cant-find - IPHLPAPI.dll was our issue.
+    # http://www.py2exe.org/index.cgi/OverridingCriteraForIncludingDlls
     options={"py2exe": {"packages": packages,
                         "optimize": 2,
                         "skip_archive": True,
                         "dist_dir": os.path.join("dist", "installdir"),
-                        "dll_excludes": ["mswsock.dll", 
-										"MSVCR90.DLL"]
+                        "dll_excludes": ["mswsock.dll", "MSVCR90.dll", "msvcr71.dll",
```

Comment:
I just noticed that `MSVCR90.dll` is excluded. Is `MSVCR90.dll` shipped with py2exe? If I remember correctly, we are also providing a Microsoft Visual C++ redistributable installer with the NSIS installer (and I thought that was executed after the installation finished).

Also, why do we exclude both `MSVCR90.dll` and `msvcr71.dll` (`msvcr71.dll` is a rather old version)?